### PR TITLE
Adding a new project page

### PR DESCRIPTION
**Pull request by kotsuki-shunji**
Initial commit for our new project.

### CODE REVIEW

File: _projects/mac_project.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+layout: page_project
+title: Monitoring the Arctic Climate
+date: 2018-03-08
+updated: 2018-03-08
+navbar: Research
+subnavbar: Projects
+project_url:
+status:
```
Comment (pancetta) [inline _projects/mac_project.md]: Please fill in the status of the project. I assume it's "starting"?

File: _projects/mac_project.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+topics:
+  - apps
+keywords:
+  -
+head:
```
Comment (pancetta) [inline _projects/mac_project.md]: Please provide the head of the project. Look at the [database](https://github.com/JLESC/jlesc.github.io/blob/source/_data/people.yml) of JLESC researchers to see if the person is already registered. If not, please do so.

File: _projects/mac_project.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+members:
+  -
```
Comment (pancetta) [inline _projects/mac_project.md]: Please fill in the members of the project, again using the database (and tags!!).

**Review by pancetta — CHANGES_REQUESTED** [review]

File: _projects/mac_project.md (hunk 3 above)
Comment (kotsuki-shunji) [reply _projects/mac_project.md]: Thank you for your response.
I updated my files.

**Comment by pancetta** [comment]
Nice, good job! Thanks!

**Pull request merged**